src/pages/404.js

```diff
@@ -40,6 +40,15 @@ const ButtonWrapper = styled.div`
 `
 
 const PageNotFound = () => {
+    React.useEffect(() => {
+        if (window.location.pathname.includes('/regulatory/kid')) {
+            navigate(
+                window.location.origin +
+                    window.location.pathname.slice(0, 16) +
+                    window.location.pathname.slice(19),
+            )
+        }
+    }, [])
```

Comment:
Hey @mohammad-rework, I think the fix here for that missing link would be to provide the PDF for that language in the directory. For this method, the client would still see the 404 screen before being routed back to the English version.